src/components/atoms/InputErrorMessage/InputErrorMessage.tsx

```diff
@@ -0,0 +1,10 @@
+import {Text} from 'react-native';
+import {styles} from './styles';
+
+type TInputErrorMessageProps = {
+  children: string;
+};
+
+export const InputErrorMessage = ({children}: TInputErrorMessageProps) => {
+  return <Text style={styles.message}>{children}</Text>;
```

Comment:
Polecam se theme całe zrobić. Swój własny komponent Text, który będzie przyjmował typ (np. enum "primary", "secondary", itp. i ustal który jaki ma fontWeight i color), to wtedy ułatwi mocno

Reply:
done